tests/many_to_many/tests.py

```diff
@@ -165,6 +166,116 @@ def test_slow_add_ignore_conflicts(self):
                 self.a1.publications.add(self.p1)
         mocked.assert_called_once()
 
+    def test_add_relations(self):
+        self.assertIn(self.p1, self.a1.publications.all())
```

Comment:
`self.assertQuerysetEqual(self.a1.publications.all(), [self.p1])` is a way to remove these repetitive assertions in all the tests below